CFUValidator/Validators/ZipContentsValidator.cs

```diff
@@ -72,8 +72,8 @@ private void DownloadPatchFile(XenServerPatchAlert patch)
 
             string tempFileName = NewTempPath();
             DownloadAndUnzipXenServerPatchAction action = new DownloadAndUnzipXenServerPatchAction(patch.Patch.Name, 
-                                                                    new Uri(patch.Patch.PatchUrl), 
-                                                                    tempFileName);
+                                                                    new Uri(patch.Patch.PatchUrl),
+                                                                    tempFileName, false, new string[] {"iso"});

```

Comment:
The patch extension can be "xsupdate" or "iso", depending on the server version. 
If we pass both extensions to the action, it will succeed in finding a patch in the archive, regardless of the server version that the patch is applicable to. E.g.:
```
DownloadAndUnzipXenServerPatchAction(patch.Patch.Name, 
    new Uri(patch.Patch.PatchUrl),
    tempFileName, false, Branding.Update, Branding.UpdateIso); 
````